app/org/maproulette/controllers/api/TaskReviewController.scala

```diff
@@ -0,0 +1,223 @@
+// Copyright (C) 2019 MapRoulette contributors (see CONTRIBUTORS.md).
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+package org.maproulette.controllers.api
+
+import javax.inject.Inject
+import org.maproulette.Config
+import org.maproulette.data.ActionManager
+import org.maproulette.models.dal._
+import org.maproulette.models.{Challenge, Task}
+import org.maproulette.permissions.Permission
+import org.maproulette.session.{SessionManager, SearchParameters, User}
+import org.maproulette.exception.{InvalidException, NotFoundException}
+import org.maproulette.utils.Utils
+import play.api.libs.json._
+import play.api.libs.ws.WSClient
+import play.api.mvc._
+
+/**
+  * TaskReviewController is responsible for handling functionality related to
+  * task reviews.
+  *
+  * @author krotstan
+  */
+class TaskReviewController @Inject()(override val sessionManager: SessionManager,
```

Comment:
Nice! Thanks for moving this into another class. The TaskController was getting a bit crowded.